backend/app/tests/api/routes/configs/test_config.py

```
@@ -59,6 +59,43 @@ def test_create_config_success(
     )
 
 
+def test_create_config_with_guardrails_persists_validator_refs(
+    db: Session,
```

Ruff (0.15.0)

[warning] Unused function argument: `db`

(ARG001)